tests/filecheck/func_ops.xdsl

```diff
@@ -32,4 +32,8 @@ builtin.module() {
   // CHECK-NEXT:   func.return(%{{.*}} : !i32)
   // CHECK-NEXT: }
 
+  func.func() ["sym_name" = "external_fn", "function_type" = !fun<[!i32], [!i32]>, "sym_visibility" = "private"] { }
+
+  // CHECK: func.func() ["sym_name" = "external_fn", "function_type" = !fun<[!i32], [!i32]>, "sym_visibility" = "private"] {}
```

Comment:
(This is heavily connected to my other comment) I think this creates a region without any blocks, so this is not quite the same thing as what gets constructed when the constructor is called. Maybe we should make to test the same thing as the thing that the constructor creates? Though that would not be possible through filecheck, AFAIU.